### ISSUE TITLE
FR: Interleaved data from two tables

### ISSUE THREAD

**Issue by hummelstrand** [jstarpl/nightscout-exporter#7]
Feature request: I just realized it's only pulling from the "entries" table. The "treatments" table is also needed. The best would be for the default export to include the date-sorted data from both tables, interleaved into the export.

In the Advanced section it might be useful for the user to be able to select (with checkboxes) if you want "Blood Glucose Readings" and/or "Treatments".

**Comment by hummelstrand** [comment]
@jstarpl Was this fixed before the pull request to the main Nightscout repo? It doesn't seem that the "treatments" table is when I just tried https://jstarpl.github.io/nightscout-exporter/

**Comment by jstarpl** [comment]
No, I fixed that in the PR to the main Nightscout repo only. It turned out I didn't fully understand the data coming in and the original version (the one on GH pages) just pulled the wrong data.
